core/templates/dev/head/pages/story-editor-page/editor-tab/story-node-editor.directive.ts

```diff
@@ -95,6 +99,8 @@ angular.module('oppia').directive('storyNodeEditor', [
             $scope.editableOutline = $scope.getOutline();
             $scope.explorationId = $scope.getExplorationId();
             $scope.currentExplorationId = $scope.explorationId;
+            $scope.expIdIsValid = true;
+            $scope.showInvalidExpError = false;
```

Comment:
For booleans, start the names with a noun: invalidExpErrorIsShown.

Only methods/functions should have names starting with a verb. (Cf. other examples in this file.)

Reply:
Done!